README.md

```diff
@@ -1 +1,198 @@
 # gazebo_ros2_control
+
+This is a ROS 2 package for integrating the `ros2_control` controller architecture with the [Gazebo](http://gazebosim.org/) simulator.
+
+This package provides a Gazebo plugin which instantiates a `ros2_control` controller manager and connects it to a Gazebo model.
+
+# Usage
+
+## Add transmission elements to a URDF
+
+To use `ros2_control` with your robot, you need to add some additional elements to your URDF. The ``<transmission>`` element is used to link actuators to joints, see the `<transmission>` spec for exact XML format.
+
+For the purposes of `gazebo_ros2_control` in its current implementation, the only important information in these transmission tags are:
+
+ - `<joint name="">` the name must correspond to a joint else where in your URDF
+ - `<type>` the type of transmission. Currently only `transmission_interface/SimpleTransmission` is implemented.
+ - `<hardwareInterface>` within the `<actuator>` and `<joint>` tags, this tells the `gazebo_ros2_control` plugin what hardware interface to load (position, velocity or effort interfaces).
+
+## Add the gazebo_ros2_control plugin
+
+In addition to the transmission tags, a Gazebo plugin needs to be added to your URDF that
+actually parses the transmission tags and loads the appropriate hardware interfaces and
+controller manager. By default the `gazebo_ros2_control` plugin is very simple, though it is also
+extensible via an additional plugin architecture to allow power users to create their own custom
+robot hardware interfaces between `ros2_control` and Gazebo.
+
+```xml
+<gazebo>
+    <plugin filename="libgazebo_ros2_control.so" name="gazebo_ros2_control">
+      <robotSimType>gazebo_ros2_control/DefaultRobotHWSim</robotSimType>
+      <robotParamNode>robot_state_publisher</robotParamNode>
+      <legacyModeNS>True</legacyModeNS>
```

Comment:
Legacy can be removed.

Also, now that we're using `gazebo_ros::Node`, you could add some examples inside `<plugin><ros>`, like namespaces and remappings.

Reply:
ping @ahcorde

Reply:
Added `<remapping>` and `<namespace>`  https://github.com/ros-simulation/gazebo_ros2_control/pull/12/commits/b8ce19264941152840122f633fac8b954632e0a4